connect/connect-core/connect-json/schema-registry/src/main/java/io/miragon/miranum/connect/json/registry/adapter/out/SchemaPersistenceAdapter.java

```diff
@@ -0,0 +1,39 @@
+package io.miragon.miranum.connect.json.registry.adapter.out;
+
+import io.miragon.miranum.connect.json.registry.application.ports.out.LoadSchemaPort;
+import io.miragon.miranum.connect.json.registry.application.ports.out.SaveSchemaPort;
+import io.miragon.miranum.connect.json.registry.domain.Schema;
+import io.miragon.miranum.connect.json.registry.domain.SchemaNew;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Repository;
+
+import java.util.Optional;
+
+@Repository
+@RequiredArgsConstructor
+public class SchemaPersistenceAdapter implements SaveSchemaPort, LoadSchemaPort {
+
+    private final SchemaRepository schemaRepository;
+    private final SchemaEntityMapper schemaEntityMapper;
+
+    @Override
+    public void saveSchema(final SchemaNew schemaNew) {
```

Comment:
There is no SchemaNew class.
There could be a Command in the Use Case or 2 Parameters in this Method.